net/ddns-scripts/files/usr/lib/ddns/dynamic_dns_functions.sh

```diff
@@ -745,6 +745,13 @@ do_transfer() {
 		# disable proxy if no set (there might be .wgetrc or .curlrc or wrong environment set)
 		[ -z "$proxy" ] && __PROG="$__PROG --no-proxy"
 
+		# user agent string if provided
+		if [ ! -z "$user_agent" ]; then
```

Comment:
We could then use `-n` here then we wouldn't have to invert `-z`?
Can you verify, whether it works like that as I suggested?